config-as-code/environments/ng-health-training.yaml

```
@@ -0,0 +1,786 @@
+#   persistence:
+#     enabled: true
+#     aws:
+#       - volumeId: "" ## add your respective volume id eg. vol-0085073c65f76dre5
+#         zone: af-south-1b            ## add your respective zone eg. ap-south-1b
+#       - volumeId: "" ## add your respective volume id eg. vol-0085073c65f76dre5
+#         zone: af-south-1b            ## add your respective zone eg. ap-south-1b
+#       - volumeId: "" ## add your respective volume id eg. vol-0085073c65f76dre5
+#         zone: af-south-1b            ## add your respective zone eg. ap-south-1b
+
+  ## Zookeeper JVM Heap Option
+  heapOptions: "-Xms256M -Xmx256M"
```

YAMLlint (1.37.1)

[error] duplication of key "heapOptions" in mapping

(key-duplicates)